Update list of supported packages in Fusion (#7881)
fishtown-analytics/docs.getdbt.com: website/snippets/_fusion-supported-packages.md

- [fivetran/google_ads](https://github.com/fivetran/dbt_google_ads.git)
- [fivetran/hubspot](https://github.com/fivetran/dbt_hubspot.git)
- [fivetran/jira](https://github.com/fivetran/dbt_jira.git)
- [fivetran/linkedin](https://github.com/fivetran/dbt_linkedin.git)
- [fivetran/microsoft_ads](https://github.com/fivetran/dbt_microsoft_ads.git)
- [fivetran/pendo](https://github.com/fivetran/dbt_pendo.git)
- [fivetran/qualtrics](https://github.com/fivetran/dbt_qualtrics.git)
- [fivetran/salesforce](https://github.com/fivetran/dbt_salesforce.git)
- [fivetran/salesforce_formula_utils](https://github.com/fivetran/dbt_salesforce_formula_utils.git)
- [fivetran/social_media_reporting](https://github.com/fivetran/dbt_social_media_reporting.git)
- [fivetran/zendesk](https://github.com/fivetran/dbt_zendesk.git)
- [GJMcClintock/dbt_tld](https://github.com/GJMcClintock/dbt_tld.git)
- [godatadriven/dbt_date](https://github.com/godatadriven/dbt-date.git)
- [metaplane/dbt_expectations](https://github.com/metaplane/dbt-expectations.git)
- [Montreal-Analytics/snowflake_utils](https://github.com/Montreal-Analytics/dbt-snowflake-utils.git)

> In addition, when making their packages compatible with <Constant name="fusion_engine" />, the Fivetran team consolidated their source and transformation packages into a single one. If you were manually installing source packages like `fivetran/github_source` but not using the `fivetran/github` package, you will now need to install `fivetran/github` and deactivate the transformation models.

Reviewer: ```suggestion
Additionally, the Fivetran `source` and `transformation` packages have been combined into a single package. If you manually installed source packages like `fivetran/github_source`, you need to ensure `fivetran/github` is installed and deactivate the transformation models.
```
